api/types/types.go

```diff
@@ -125,6 +126,11 @@ type RubyResults struct {
 	HuskyCIBrakemanOutput HuskyCISecurityTestOutput `bson:"brakemanoutput,omitempty" json:"brakemanoutput,omitempty"`
 }
 
+// GitResults represents all Git security tests results.
```

Comment:
If you would like to follow my suggestion of using `GenericResullts` here, we could use the following comment: `GenericResults represents all generic securityTests results.`

Reply:
sure, that sounds like a good idea,
so instead of GitResults, we gonna have GenericResults right?

Reply:
Yep! If you agree with this suggestion, of course!  🙃🚀

Reply:
i do agree sir 👍